addons/hr_recruitment/data/scenarios/hr_recruitment_scenario.xml

```diff
@@ -320,14 +306,7 @@
             <field name="date" eval="DateTime.today() - relativedelta(days=10)" />
             <field name="author_id" ref="base.user_admin" />
             <field name="subtype_id" ref="hr_recruitment.mt_applicant_stage_changed" />
```

Comment:
we need a
`
    <field name="message_type">tracking</field>
`

Reply:
So now we need a special and separate message for tracking, so more records?

Reply:
Not a separate message just a new `message_type` to classify messages of type "track values". That way, we can do ACL, client side formatting, ... bassed on this message_type.